etc/qgis.profile

```diff
@@ -0,0 +1,46 @@
+# Firejail profile for qgis
+# Description: GIS application
+# This file is overwritten after every install/update
+# Persistent local customizations
+include qgis.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/QtProject.conf
+
+# Allow python (blacklisted by disable-interpreters.inc)
+noblacklist ${PATH}/python3*
+noblacklist /usr/lib/python3*
+noblacklist /usr/local/lib/python3*
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+whitelist ${HOME}/.local/share/QGIS/
+include whitelist-common.inc
+include whitelist-var-common.inc
+
+caps.drop all
+netfilter
+nodbus
+nodvd
+nogroups
+nonewprivs
+noroot
+nosound
+notv
+nou2f
+novideo
+seccomp
```

Comment:
qgis don't start with `seccomp` on my system.

Reply:
> qgis don't start with `seccomp` on my system.

interesting, it does work for me on a fresh arch linux system

Reply:
the default list without `mbind` works:
```
seccomp.drop _sysctl,acct,add_key,adjtimex,afs_syscall,bdflush,bpf,break,chroot,clock_adjtime,clock_settime,create_module,delete_module,fanotify_init,finit_module,ftime,get_kernel_syms,getpmsg,gtty,init_module,io_cancel,io_destroy,io_getevents,io_setup,io_submit,ioperm,iopl,ioprio_set,kcmp,kexec_file_load,kexec_load,keyctl,lock,lookup_dcookie,migrate_pages,modify_ldt,mount,move_pages,mpx,name_to_handle_at,nfsservctl,ni_syscall,open_by_handle_at,pciconfig_iobase,pciconfig_read,pciconfig_write,perf_event_open,personality,pivot_root,process_vm_readv,process_vm_writev,prof,profil,ptrace,putpmsg,query_module,reboot,remap_file_pages,request_key,rtas,s390_mmio_read,s390_mmio_write,s390_runtime_instr,security,set_mempolicy,setdomainname,sethostname,settimeofday,sgetmask,ssetmask,stime,stty,subpage_prot,swapoff,swapon,switch_endian,sys_debug_setcontext,sysfs,syslog,tuxcall,ulimit,umount,umount2,uselib,userfaultfd,ustat,vhangup,vm86,vm86old,vmspliceandvserver
```
For me: Fedora29 with qgis 2.18.20
qgis current: 3.6.X
but for example debian buster still has qgis 2.18.28.
https://packages.debian.org/buster/qgis